tests/integration/integration_steps/fabric_steps.py

```diff
@@ -12,10 +16,25 @@ def read_deltalake_timeseries(timeseries_path:str, credential: DefaultAzureCrede
     df = delta_table.to_pandas()
     return df
 
+def prepare_lakehouse_dataframe_for_comparison(dataframe, external_id):
+    dataframe = dataframe.loc[dataframe["externalId"] == external_id]
+    if dataframe[TIMESTAMP_COLUMN].dt.tz is None:
+        local_tz = tz.tzlocal()
+        dataframe[TIMESTAMP_COLUMN] = dataframe[TIMESTAMP_COLUMN].dt.tz_localize(local_tz)
```

Comment:
Could trigger the warning like the following:
<img width="1052" alt="Screenshot 2024-03-22 102418" src="https://github.com/kenny-chalupa/cdf-fabric-replicator/assets/123187956/b1a5b8b7-7a78-44b4-a0ed-14737a11bef1">
Suggested change:
`dataframe.loc[:,TIMESTAMP_COLUMN] = dataframe[TIMESTAMP_COLUMN].dt.tz_localize(local_tz)`.

Reply:
I tried this and I got attribute errors for trying to use datetime operators on non datetime objects.  I'm also not getting this warning on my end

Reply:
I got it probably from an older version of the branch. It's OK to leave as is and if the warning comes back we can think about how to resolve it.